crates/unleash-edge/src/edge_builder.rs

```diff
@@ -15,16 +15,20 @@ use unleash_edge_delta::cache_manager::DeltaCacheManager;
 use unleash_edge_feature_cache::FeatureCache;
 use unleash_edge_feature_refresh::{FeatureRefreshConfig, FeatureRefresher};
```

Comment:
The logic change in this file is just to assemble the background task list and bubble it to the caller